pytensor/link/pytorch/dispatch/basic.py

```diff
@@ -132,3 +134,17 @@ def makevector(*x):
         return torch.tensor(x, dtype=torch_dtype)
 
     return makevector
+
+
+@pytorch_funcify.register(OpFromGraph)
+def pytorch_funcify_OpFromGraph(op, node=None, **kwargs):
+    _ = kwargs.pop("storage_map", None)
+
+    PYTORCH.optimizer(op.fgraph)
+    fgraph_fn = torch.compile(pytorch_funcify(op.fgraph, **kwargs))
+
+    def opfromgraph(*inputs, dim=op.fgraph.outputs):
```

Comment:
What is dim?

Reply:
this is currently unused, and typo'd. It should be `len(op.fgraph.outputs)`, and we'll just check that to ensure we return the correct sized value.

Reply:
fixed